OneMore/Commands/Navigator/NaviPagesCommand.cs

```diff
@@ -0,0 +1,130 @@
+﻿//************************************************************************************************
+// Copyright © 2020 Steven M Cohn.  All rights reserved.
+//************************************************************************************************
+
+namespace River.OneMoreAddIn.Commands
+{
+    using River.OneMoreAddIn.Commands.Navigator;
+    using River.OneMoreAddIn.Commands.Search;
+    using System;
+    using System.Collections.Generic;
+    using System.Security.Policy;
+    using System.Threading.Tasks;
+    using System.Windows.Forms;
+    using Resx = River.OneMoreAddIn.Properties.Resources;
+    using System.Xml.Linq;
+    using System.Linq;
+
+
+
+
+    internal class NaviPagesCommand : Command
+    {
+        private bool copying;
+        private List<string> pageIds;
+
+        public NaviPagesCommand()
+        {
+            
+
+        }
+
+
+        public override async Task Execute(params object[] args)
+        {
+            logger.WriteLine("command starts");
+            copying = false;
+
+            var dialog = new NaviPages();
+            await dialog.RunModeless((sender, e) =>
+            {
+                var d = sender as NaviPages;
+                if (d.DialogResult == DialogResult.OK)
+                {
+                    //copying = dialog.CopySelections;
+                    //pageIds = dialog.SelectedPages;
+                    ////DestPgID = dialog.DestPageID;
+
+                    //var action = copying ? "copying" : "moving";
+
+                    //var desc = copying
+                    //    ? Resx.SearchQF_DescriptionCopy
+                    //    : Resx.SearchQF_DescriptionMove;
+
+                    //using var one = new OneNote();
+                    //one.SelectLocation(Resx.SearchQF_Title, desc, OneNote.Scope.Sections, Callback);
+
+                    try
+                    {
+                        using var one = new OneNote();
+                        //var service = new SearchServices(owner, one, sectionId);
+
+                        if (copying)
```

Comment:
Are these operations going to be implemented? If not, remove the code and associated UI controls from the dialog